src/convert.ts

```diff
@@ -16,17 +17,34 @@ export const defaultArgs = [
 
 const INPUT_PATH = '/opt/lo.tar.br';
 const OUTPUT_PATH = '/tmp/instdir/program/soffice.bin';
+const UNOPKG_OUTPUT_PATH = '/tmp/instdir/program/unopkg.bin';
 
 /**
  * Converts a file in /tmp to the desired file format
  * @param {String} filename Name of the file to convert located in /tmp directory
  * @param {String} format File format to convert incoming file to
+ * @param {Array} extensions List of LibreOffice extension paths (.oxt files)
  * @return {Promise<String>} Absolute path to the converted file
  */
-export async function convertTo(filename: string, format: string): Promise<string> {
+export async function convertTo(
+  filename: string,
+  format: string,
+  extensions: string[] = []
+): Promise<string> {
   let logs;
   cleanupTempFiles();
   await unpack({inputPath: INPUT_PATH});
+
+  if (extensions.length > 0) {
+    const enabledExtensions = execSync(`${UNOPKG_OUTPUT_PATH} list --shared`);
```

Comment:
should it be wrapped in try-catch, to not produce fail if the extension could not be loaded?

Reply:
Sure, that can be done

Reply:
What do you think about extending input array to -> 
{extensions = [], shouldThrowOnExtensionFail = false}
and make possibility still to throw out of converting if critical extension not loaded

Reply:
I wonder if it should be set to false by default though, perhaps we do want the exceptions to be thrown if expected extensions could not be enabled by default ?